monitor/sth_fetcher.go

```diff
@@ -21,6 +21,7 @@ type sthFetchStats struct {
 	sthTimestamp       *prometheus.GaugeVec
 	sthAge             *prometheus.GaugeVec
 	sthFailures        *prometheus.CounterVec
+	sthFetchTotal      *prometheus.CounterVec
```

Comment:
Can you also add this new metric to the README: https://github.com/letsencrypt/ct-woodpecker/blob/master/README.md#collected-metrics